markdown_it/extensions/tasklists/__init__.py

```diff
@@ -0,0 +1,159 @@
+"""Builds task/todo lists out of markdown lists with items starting with [ ] or [x]"""
+
+# Ported by Wolmar Nyberg Åkerström from https://github.com/revin/markdown-it-task-lists
+# ISC License
+# Copyright (c) 2016, Revin Guillen
+#
+# Permission to use, copy, modify, and/or distribute this software for any
+# purpose with or without fee is hereby granted, provided that the above
+# copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+# WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+# MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+# ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+# WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+# ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+# OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+from math import ceil
+from random import random
+from typing import List
+
+from markdown_it import MarkdownIt
+
+from markdown_it.token import Token
+
+
+def tasklists_plugin(
+    md: MarkdownIt,
+    enabled: bool = False,
+    label: bool = False,
+    label_after: bool = False,
+):
+    """Plugin for building task/todo lists out of markdown lists with items starting with [ ] or [x]
+    .. Nothing else
+
+    The rendered checkboxes are disabled; to change this, pass a truthy value into the enabled
+    property of the plugin options.
+
+    :param enabled: True enables the rendered checkboxes
```

Comment:
it wasn't really clear to me what enable meant in the current description

```suggestion
    :param enabled: True adds an "enabled" CSS class to each list item 
```

Reply:
oh no actually maybe this was a misunderstanding by me.

I see it adds `disabled="disabled"` to the `input` markup, which I guess means it is frozen and you can't toggle its value

Reply:
Yepp, it's the ability to change the values of the checkboxes. I ported the documentation with the phasing mostly intact.

Reply:
There is a comment on this option in the general part of the docstring